packages/app-mobile/components/global-style.ts

```diff
@@ -112,6 +117,9 @@ function extraStyles(theme: BaseTheme) {
 		keyboardAppearance: theme.appearance,
 		color5: theme.color5 ?? theme.backgroundColor4,
 		backgroundColor5: theme.backgroundColor5 ?? theme.color4,
+
+		backgroundColorHover4: Color(theme.color4).alpha(0.12).rgb().string(),
```

Comment:
`backgroundColorHover4` determines the ripple/hover color for the `<TouchableRipple>` menu item component. This use of `Color` matches how additional theme colors are added for the desktop app. For example,
https://github.com/laurent22/joplin/blob/d7b3ef8ddf60cd91b350ec7a750bbdf8542185cc/packages/lib/theme.ts#L111

Additionally, `.alpha(0.12)` [should match React Native's `<Button>` hover color](https://github.com/callstack/react-native-paper/blob/9162a1fdf4e97c0669de8a8fa2057d8cfbe2e728/src/components/Button/Button.tsx#L281).